components/expenses/ExpenseForm.js

```diff
@@ -77,15 +77,19 @@ const msg = defineMessages({
     defaultMessage: 'Payee information',
   },
   cancelEditExpense: {
+    id: 'ExpenseForm.CancelEditExpense',
     defaultMessage: 'Cancel Edit',
   },
   confirmCancelEditExpense: {
+    id: 'ExpenseForm.ConfirmCancelEditExpense',
     defaultMessage: 'Are you sure you want to cancel the edits?',
   },
   clearExpenseForm: {
+    id: 'ExpenseForm.ClearExpenseForm',
     defaultMessage: 'Clear Form',
   },
   confirmClearExpenseForm: {
+    id: 'ExpenseForm.ConfirmClearExpenseForm',
```

Comment:
Let's keep these as they were, IDs are optional and adding them will invalidate all the current translations for these strings